grove/alpha/jordan_gradient/jordan_gradient.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import division
+
+import numpy as np
+import pyquil.quil as pq
+from pyquil.gates import X, H
+from grove.alpha.phaseestimation.phase_estimation import phase_estimation
+
+from grove.alpha.jordan_gradient.gradient_utils import binary_to_real, \
+    measurements_to_bf
+
+
+def initialize_system(ancilla_qubit):
+    """Prepare initial state
+
+    :param list ancilla_qubit: Qubit of ancilla register.
+    :return: Quil program to initialize this system.
+    :rtype: Program
+    """
+
+    # ancilla qubit to plane wave state
+    ic_ancilla = pq.Program([X(ancilla_qubit), H(ancilla_qubit)])
+    p_ic = pq.Program(ic_ancilla)
+
+    return p_ic
+
```

Comment:
PEP8 two newlines between module level functions, and can you change it for other functions in this file too! (If you use an IDE like pycharm it will whine about things like this)